packages/connection-form/src/hooks/use-connect-form.ts

```diff
@@ -63,13 +63,21 @@ import { applyForceConnectionOptions } from '../utils/force-connection-options';
 import { useConnectionFormPreference } from './use-connect-form-preferences';
 import ConnectionString from 'mongodb-connection-string-url';
 
+export type ConnectionPersonalisationOptions = {
```

Comment:
I don't know if this is a can of worms I want to open, so feel free to just resolve this and not worry about it, but I think we typically use en_US spelling 😂

Reply:
Oh I just realised I was writing in *the proper English*, but I can fix it by adding typos randomly and changing `s` with `z`s. :trollface: